src/assets/js/app.ts

```diff
@@ -77,6 +78,15 @@ import SoundEffects from '@js/SoundEffects';
     sunburstSvg.style.display = 'none';
   };
 
+  /** Function to persist the state to local storage */
+  const persistNames = () => {
+    // eslint-disable-next-line no-use-before-define
```

Comment:
@icelam not sure if you care, the complexity added to the code to avoid this lint didn't seem worth it to me.